src/config/mod.rs

```diff
@@ -60,19 +60,23 @@ pub async fn load_kube_config_with(options: ConfigOptions) -> Result<Configurati
     let result = create_client_builder(options).await?;
     Ok(Configuration::new(
         result.1.cluster.server,
-        result.0.build()
+        result
+            .0
+            .build()
             .map_err(|e| Error::KubeConfig(format!("Unable to build client: {}", e)))?,
     ))
 }
 
 #[cfg(target_os = "macos")]
-fn platform_cfg_client_builder(client_builder: ClientBuilder) -> ClientBuilder {
+fn platform_cfg_client_builder(
+    client_builder: ClientBuilder,
+    ca: &openssl::x509::X509,
```

Comment:
The addition of this parameter is the main update. The rest are rustfmt changes (for the better imho, but if you disagree I will redo this commit for only the specific line updates in the 3 main spots).

Reply:
Adding the parameter is fine - we probably need to do that for the `rustls` change anyway. I started passing `reqwest::Certificate`s around instead which made that difficult.

I believe the type should be in scope as just `X509` though.

Reply:
Ok, will do.